spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/post/SendResponseFilter.java

```diff
@@ -170,13 +173,8 @@ private void writeResponse(InputStream zin, OutputStream out) throws Exception {
 		byte[] bytes = new byte[INITIAL_STREAM_BUFFER_SIZE.get()];
 		int bytesRead = -1;
 		while ((bytesRead = zin.read(bytes)) != -1) {
-			try {
-				out.write(bytes, 0, bytesRead);
-				out.flush();
-			}
-			catch (IOException ex) {
```

Comment:
Again, I'm hesitant to change something like this on a point release. We don't recommend running `/hystrix.stream` through zuul.